core/src/torii/mod.rs

```diff
@@ -347,6 +373,36 @@ async fn handle_get_configuration<W: WorldTrait>(
     .map_err(Error::Config)
 }
 
+/// Message acceptable for `POST` requests to the configuration endpoint.
+#[derive(Clone, Debug, Deserialize, Serialize, Copy)]
+pub enum PostConfiguration {
+    /// Change the maximum logging level of logger.
+    ///
+    /// # Examples
+    /// To silence all logging events that aren't `ERROR`s
+    /// `curl -X POST -H 'content-type: application/json' http://127.0.0.1:8080/configuration -d '{"ChangeLogLevel": "ERROR"}' -i`
+    ChangeLogLevel(iroha_logger::Level),
```

Comment:
I would just call it `LogLevel`. `POST` http request indicates imperative mode

Reply:
imperative *mood* but yeah, I agree.